### DISCUSSION TITLE
SDK for Java

### DISCUSSION THREAD

**Discussion by rcaballeron** [WalletConnect/walletconnect-monorepo#3452] [Developer Support] [answered]
Dear all,

At our company we've developed a server hosted wallet solution based on Java. Now we want to add support for WalletConnect. I've found SDK's and sample code for different language. My questions are:

1. Is there a Java SDK available? If so, could you please redirect me to its location?
2. Is there any Java sample code I could use as reference?

Many thanks in advance for your support.

Kind regards

**Comment by glitch-txs** [comment, marked as answer]
No Java implementation at the moment :(

**Reply by rcaballeron** [reply]
Is there any plan for supporting it in the near future? In the meanwhile, any suggestion about how could we integrate our product with WalletConnect?
